applications/tari_base_node/src/command_handler.rs

```diff
@@ -884,66 +888,104 @@ impl CommandHandler {
         });
     }
 
-    pub fn raw_stats(&self, start_height: u64, end_height: u64) {
-        let mut node = self.node_service.clone();
+    pub fn save_header_stats(
+        &self,
+        start_height: u64,
+        end_height: u64,
+        filename: String,
+        pow_algo: Option<PowAlgorithm>,
+    )
+    {
+        let db = self.blockchain_db.clone();
         self.executor.spawn(async move {
-            let mut results: Vec<(u64, usize, u64, u64, u64, PowAlgorithm)> = Vec::new();
+            let mut output = try_or_print!(File::create(&filename));
+
+            println!(
+                "Loading header from height {} to {} and dumping to file [working-dir]/{}.",
```

Comment:
Would be good to have `pow_algo` in this message too

Reply:
Done :)